src/plugins/elasticsearch/lib/create_proxy.js

```diff
@@ -24,3 +22,10 @@ module.exports = function createProxy(server, method, route, config) {
   server.route(options);
 };
 
+createProxy.createPath = function createPath(path) {
+  const pre = '/elasticsearch';
+  const sep = path[0] === '/' ? '' : '/';
+  return `${pre}${sep}${path}`;
```

Comment:
Nitpick: The code above uses `trimRight` for this, so could use `trimLeft` here, e.g.

```
const post = _.trimLeft(path, '/');
`${pre}/${post}`
```